dataplane/saiserver/routing.go

```diff
@@ -510,6 +510,8 @@ func newRoute(mgr *attrmgr.AttrMgr, dataplane switchDataplaneAPI, s *grpc.Server
 	return r
 }
 
+const routeDstMeta = 0
```

Comment:
So we always have/care only one instance?

Reply:
the PACKET_ATTRIBUTE_32 is special, its actually a map, i'm using is this const as the key to for that map. added comment